### ISSUE TITLE
bash: /usr/bin/*: Permission denied after last apt upgrade

### ISSUE THREAD

**Issue by stuart-little** [termux/termux-packages#5984]
**Problem description**

With the latest `apt upgrade` (minutes ago as I write this, Fri, Oct 30 2020 about 20:30 EST) all invocations of `/usr/bin/*` (with `*=env`, `perl`, whatever) returns

```
bash: /usr/bin/<blah>: Permission denied
```

**Steps to reproduce**

Run

```
/usr/bin/env
```

**Expected behavior**

Same output as running `env`. This is based on the fact that before today's update scripts starting with the shebang line

```
#!/usr/bin/perl
```
used to work just fine. I've run them thousands of times.

**Additional information**

```
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://termux.org/packages/ stable main
# game-repo (sources.list.d/game.list)
deb https://dl.bintray.com/grimler/game-packages-24 games stable
# sources.list.d/science.list.bak
deb https://dl.bintray.com/grimler/science-packages-24 science stable
# science-repo (sources.list.d/science.list)
deb https://dl.bintray.com/grimler/science-packages-24 science stable
# sources.list.d/game.list.bak
deb https://dl.bintray.com/grimler/game-packages-24 games stable
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/24 termux extras
Updatable packages:
All packages up to date
Android version:
10
Kernel build information:
Linux localhost 4.9.179-perf+ #1 SMP PREEMPT Sun Oct 4 22:59:52 CST 2020 aarch64 Android
Device manufacturer:
OnePlus
Device model:
ONEPLUS A6013
```

**Comment by stuart-little** [comment]
To supplement that, I have checked that I have `termux-exec` installed. Additionally, [this](https://wiki.termux.com/wiki/Termux-exec) advice does not resolve the matter.

**Comment by xeffyr** [comment]
Fixed in `termux-exec` 0.8.
